config/environments/production.rb

```diff
@@ -31,7 +31,7 @@
 
   # Disable Rails's static asset server
   # In production, Apache or nginx will already do this
-  config.serve_static_assets = false
+  config.serve_static_assets = true
```

Comment:
I just realized that we might not want to do this directly, since in many deployment situations you won't want this to be `true`. We could define an environment variable for this (e.g. `RES_HEROKU` and use that to determine the setting.